ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/representative-journey/services/representative.journey.service.ts

```diff
@@ -0,0 +1,36 @@
+import { Injectable } from '@angular/core';
+import { SessionStorage } from '../../shared/services/session-storage';
+import { RepresentativeSuitabilityModel } from '../representative-suitability.model';
+import {Hearing} from '../../base-journey/participant-suitability.model';
+import {MutableRepresentativeSuitabilityModel} from '../mutable-representative-suitability.model';
+
+@Injectable()
+export class RepresentativeJourneyService {
+  private readonly cache: SessionStorage<RepresentativeSuitabilityModel>;
+
+  constructor() {
+    this.cache = new SessionStorage<RepresentativeSuitabilityModel>('REPRESENTATIVEJOURNEY_MODEL');
+  }
+
+  get(): RepresentativeSuitabilityModel {
+    const response = this.cache.get();
+
+    if (response === null) { return null; }
+
+    const model = new MutableRepresentativeSuitabilityModel();
```

Comment:
might be worthwhile to do a quick comment on why you need to do this despite the `response` seemingly being an instance of `RepresentativeSuitabilityModel` already. you and I know what the issue was about but a few weeks down the road we'll be like "what's this all about?"

Reply:
done